Tests/SonarQube.TeamBuild.Integration.Tests/CoverageReportProcessorTests.cs

```diff
@@ -1,182 +0,0 @@
-﻿using System;
-using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
-using SonarQube.Common;
-using TestUtilities;
-
-namespace SonarQube.TeamBuild.Integration.Tests
```

Comment:
What happened to the class that this tested?

Reply:
This is part of the refactoring change I've reverted.